### DISCUSSION TITLE
mqtt commands

### DISCUSSION THREAD

**Discussion by sandy1981Nett** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#100] [Q&A]
Hello,

mqtt works :-) 

I have a few questions...

clear command queue = clears the time-honored commands?
what is the command GETTARGET (internal use)?
there is no command for standby mode (button in the middle) 

Christian

**Comment by visualapproach** [comment]
> Hello,
> 
> mqtt works :-)
> 
> I have a few questions...
> 
> clear command queue = clears the time-honored commands?

Yes, the list of queued commands can be seen and programmed on the config page. And erased.

> what is the command GETTARGET (internal use)?

At start-up the target temperature is not known. Therefore we simulate button presses to find out.

> there is no command for standby mode (button in the middle)

No. It doesn't do anything but dimming the display. Other than turning stuff off, but you can do that by turning filter pump off. Actually there is an internal command to lock/power up/down the device, but people thought it was just an annoyance to do it manually, so it's done automatically.
> 
> Christian

**Reply by sandy1981Nett** [reply]
thank you! 

for iobroker / GER used mqtt look here

https://forum.iobroker.net/topic/46464/aus-json-datenpunkte-erzeugen-whirpool-anbindung/30?_=1626787160451

Chris
